src/bucket/LiveBucket.h

```diff
@@ -86,15 +86,11 @@ class LiveBucket : public BucketBase<LiveBucket, LiveBucketIndex>,
     size_t getMaxCacheSize() const;
 #endif
 
-    // Returns [lowerBound, upperBound) of file offsets for all offers in the
-    // bucket, or std::nullopt if no offers exist
+    // Returns [lowerBound, upperBound) of file offsets for all entries of the
```

Comment:
This interface seems suspicious to me as we provide multiple types, but get only a single range as an output. Does that mean that we may get some entries of types that were not supplied in the input `types`?

Reply:
Thanks for the update